pyproject.toml

```
@@ -38,6 +30,16 @@ cu12 = [
     "nvidia-cuda-nvrtc-cu12",
     "nvidia-nvjitlink-cu12"
 ]
+cu13 = [
+    "cuda-bindings==13.0.*",
+    "cuda-core==0.3.*",
+    "cuda-python==13.0.*",  # supports all CTK 13.x
+    "nvidia-cuda-nvcc==13.*",  # for libNVVM
```

In CUDA 13 there is now an `nvidia-nvvm` package

```suggestion
    "nvidia-nvvm==13.*",
```